egs/librispeech/ASR/pruned_transducer_stateless5/train.py

```diff
@@ -655,7 +655,35 @@ def compute_loss(
             am_scale=params.am_scale,
             lm_scale=params.lm_scale,
             warmup=warmup,
+            reduction="none",
         )
+        simple_loss_is_finite = torch.isfinite(simple_loss)
+        pruned_loss_is_finite = torch.isfinite(pruned_loss)
+        is_finite = simple_loss_is_finite & pruned_loss_is_finite
+        if not torch.all(is_finite):
+            logging.info(
+                "Not all losses are finite!\n"
+                f"simple_loss: {simple_loss}\n"
+                f"pruned_loss: {pruned_loss}"
+            )
+            display_and_save_batch(batch, params=params, sp=sp)
+            simple_loss = simple_loss[simple_loss_is_finite]
```

Comment:
Just a heads up: I was facing the same logical indexing issue here that we had [seen earlier](https://github.com/k2-fsa/icefall/issues/240#issuecomment-1065402350), so I had to replace this with the `torch.where()`. I suppose this might just be a CUDA 11.1 issue after all.